scico/linop/xray/_util.py

```diff
@@ -173,6 +183,8 @@ def image_alignment_rotation(
 
 
     """
+    if not have_astra:
+        raise RuntimeError("Package astra is required for use of this function.")
```

Comment:
Consider simply `import astra` here without any of the other mechanism above.

Reply:
The goal is to make `astra` an optional dependency. This mechanism is in place because there's only one function in the module that uses `astra`.

Reply:
Placing `import astra` in the body of that one function accomplishes the goal of making astra optional in a single line, whereas the current mechanism uses 10+ lines.